fiorella/src/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,15 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#E83671, 0.7);
```

Comment:
Color literals like `#E83671` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E83671` should be written as `#e83671`